### DISCUSSION TITLE
Map of DARC DOK

### DISCUSSION THREAD

**Discussion by nobswolf** [wavelog/wavelog#635] [Show and tell]
I am the author of the "dokmap", a map based on OSM and Leaflet with some help from the DARC head quarters. You find it in Wavelog in the award-menu of Germany linked as "map". 

If you have any questions, feature requests or issues about dokmap with respect to Wavelog we can discus this here.

**Comment by AndreasK79** [comment]
Hi and welcome. It's great to have you here. If anything regarding the map pops up, we'll let you know.

If you want to contribute in Wavelog, we are thankful for any contribution.

Andreas.
